packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -134,40 +149,28 @@ contract PerpetualPositionManager is FundingRatePayer, FundingRateApplier {
      * We recommend to only use synthetic token contracts whose sole Owner role (the role capable of adding & removing roles)
      * is assigned to this contract, whose sole Minter role is assigned to this contract, and whose
      * total supply is 0 prior to construction of this contract.
-     * @param _withdrawalLiveness liveness delay, in seconds, for pending withdrawals.
-     * @param _collateralAddress ERC20 token used as collateral for all positions.
-     * @param _tokenAddress ERC20 token used as synthetic token.
-     * @param _finderAddress UMA protocol Finder used to discover other protocol contracts.
-     * @param _priceIdentifier registered in the DVM for the synthetic.
-     * @param _fundingRateIdentifier Unique identifier for DVM price feed ticker for child financial contract.
-     * @param _fundingRateRewardRate Reward rate to pay FundingRateStore to use funding rate data.
-     * @param _minSponsorTokens minimum amount of collateral that must exist at any time in a position.
-     * @param _timerAddress Contract that stores the current time in a testing environment. Set to 0x0 for production.
-     * @param _excessTokenBeneficiary Beneficiary to send all excess token balances that accrue in the contract.
+     * @param params struct to define input parameters for construction of PositionManager.
      */
-    constructor(
```

Comment:
this is the first major interface deviation between the emp and the perp as now we feed in a struct. (clearly, the params are different between the contracts but at least the structure is consistent).

Reply:
yeah I can remove this struct for now

Reply:
done